src/azul/tdr.py

```diff
@@ -169,6 +186,34 @@ def __init__(self, dataset: BigQueryDataset):
         self.target = dataset
         self.big_query_adapter.assert_table_exists(dataset.name, 'links')
 
+    @classmethod
+    @lru_cache
+    def oauthed_http(cls) -> AuthorizedHttp:
+        """
+        A urllib3 HTTP client with OAuth credentials.
+        """
+        with shared_credentials():
+            credentials = Credentials.from_service_account_file(os.environ['GOOGLE_APPLICATION_CREDENTIALS'])
+        return AuthorizedHttp(credentials.with_scopes(['email', 'openid']),
+                              urllib3.PoolManager(ca_certs=certifi.where()))
+
+    @classmethod
+    def verify_authorization(cls) -> None:
+        """
+        Raise RequirementError if an authorization error occurs during a simple API call to TDR.
+        """
+        # List snapshots
+        response = cls.oauthed_http().request('GET',
+                                              f'{config.tdr_service_url}/api/repository/v1/snapshots')
+        if response.status == 200:
+            log.info('Google service account is authorized for TDR access.')
+        elif response.status == 401:
+            raise RequirementError('Google service account is not authorized for TDR access.'
+                                   ' Make sure that the SA is registered with SAM and has been'
```

Comment:
Nowhere in the code base do we wrap strings before the space like this. Please stick with precedent or propose a change.